### DISCUSSION TITLE
Simple simulator

### DISCUSSION THREAD

**Discussion by EricCabrol** [thomasfermi/Algorithms-for-Automated-Driving#8] [Q&A] [answered]
Hello

I am quite impressed by the job you did there ! Congratulations, and thank you for the links to Prof. Schildbach's channel. 

You mention a "simplistic simulator" in the Controls overview page, but I didn't see any other mention, and the only solution for the moment seems to use Carla. Is it something you planned ?

**Comment by thomasfermi** [comment, marked as answer]
Hi @EricCabrol! Thanks for your kind words :)

The (very very simple) simulator is here
https://github.com/thomasfermi/Algorithms-for-Automated-Driving/blob/master/code/tests/control/simulation.py

and it is being used for the exercise in the section on Pure Pursuit. There, you are asked to work on code/tests/control/control.ipynb, which you can see here:

https://github.com/thomasfermi/Algorithms-for-Automated-Driving/blob/master/code/tests/control/control.ipynb

The simulator will be started from that notebook

**Reply by EricCabrol** [reply]
Oh thanks, I had missed it !

**Reply by thomasfermi** [reply]
No problem! You are not the first one :) On the discord server someone asked the same question. I added a hint about the simulator in the exercise description to make it clearer.
